tests/integration-tests/tests/scaling/test_mpi.py

```diff
@@ -51,3 +56,35 @@ def test_mpi(scheduler, region, os, instance, pcluster_config_reader, clusters_f
         scaledown_idletime,
         verify_scaling=True,
     )
+
+
+@timeout(5, use_signals=False, timeout_exception=RemoteCommandExecutionError)
+def _test_mpi_comms(remote_command_executor, scheduler, os):
+    logging.info("Testing mpi communications")
+    mpi_module = OS_TO_OPENMPI_MODULE_MAP[os]
+
+    scheduler_commands = get_scheduler_commands(scheduler, remote_command_executor)
+    compute_node = scheduler_commands.get_compute_nodes()
+    assert_that(len(compute_node)).is_equal_to(1)
+    remote_host = compute_node[0]
+    mpirun_out = remote_command_executor.run_remote_command(
+        "module load {0} && mpirun --host {1} hostname".format(mpi_module, remote_host), raise_on_error=False
+    ).stdout.splitlines()
+
+    # mpirun_out =
+    # "Warning: Permanently added the RSA host key for IP address '10.0.127.71' to the list of known hosts.\n
+    # ip-10-0-127-71"
+    assert_that(len(mpirun_out)).is_greater_than_or_equal_to(1)
+    assert_that(mpirun_out[-1]).is_equal_to(remote_host)
+
+
+@pytest.mark.regions(["eu-west-1"])
+@pytest.mark.instances(["c5.xlarge"])
+@pytest.mark.schedulers(["slurm", "sge", "torque"])
+@pytest.mark.oss(["alinux"])
+def test_mpirun_comms(scheduler, region, os, instance, pcluster_config_reader, clusters_factory):
```

Comment:
minor: region and instance if not explicitly referred in the code can be declared in the usefixtures annotations